sqldb/sqlc/migrations/000002_invoices.up.sql

```diff
@@ -0,0 +1,57 @@
+-- The invoices table contains all the information shared by all the invoice 
+-- types. 
+CREATE TABLE IF NOT EXISTS invoices (
+        id INTEGER PRIMARY KEY,
+
+        memo TEXT,
```

Comment:
nit: I think it's worth adding inline docs to the schema at least to some of the fields and to all tables.

Reply:
added